test/test_other.py

```diff
@@ -2470,6 +2470,7 @@ def test_sdl_get_key_name(self):
   def test_sdl2_mixer_wav(self):
     self.emcc(test_file('browser/test_sdl2_mixer_wav.c'), ['-sUSE_SDL_MIXER=2'], output_filename='a.out.js')
     self.emcc(test_file('browser/test_sdl2_mixer_wav.c'), ['--use-port=sdl2_mixer'], output_filename='a.out.js')
+    self.emcc(test_file('browser/test_sdl2_mixer_wav.c'), ['--use-port=sdl2_mixer:formats=ogg'], output_filename='a.out.js')
```

Comment:
How does this test (which plays a wav file) work if only ogg is selected?  Is this because `wav` is simply always available maybe?

Reply:
Yeah, I think so. wav is already available:

https://github.com/emscripten-core/emscripten/blob/2328a81304238c2d69ce8d1662e3f3505fe88b85/tools/ports/sdl2_mixer.py#L44-L47